### PR TITLE
support Go v1.16 version

### PR DESCRIPTION
**Pull request by thinkerou**

### CODE REVIEW

**Comment by thinkerou** [comment]
fix go master branch issue:

```
    routes_test.go:363: 
        	Error Trace:	routes_test.go:363
        	Error:      	Not equal: 
        	            	expected: "text/plain; charset=utf-8"
        	            	actual  : "text/x-go; charset=utf-8"
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-text/plain; charset=utf-8
        	            	+text/x-go; charset=utf-8
        	Test:       	TestRouterMiddlewareAndStatic
```

**Comment by thinkerou** [comment]
@appleboy please review the pr, because it hold pr merge.

File: context_test.go [hunk 1]
```diff
@@ -1018,7 +1018,9 @@ func TestContextRenderFile(t *testing.T) {
 
 	assert.Equal(t, http.StatusOK, w.Code)
 	assert.Contains(t, w.Body.String(), "func New() *Engine {")
-	assert.Equal(t, "text/plain; charset=utf-8", w.Header().Get("Content-Type"))
```
Comment (appleboy) [inline context_test.go:1021]: @thinkerou Could you provide some refernce?

Comment (thinkerou) [reply context_test.go:1021]: @appleboy https://github.com/gin-gonic/gin/pull/2638#discussion_r616274265

**Review by appleboy — APPROVED** [review]

**Pull request merged**